packages/base/src/panelview/objectproperties.tsx

```diff
@@ -186,27 +186,173 @@ class ObjectPropertiesReact extends React.Component<IProps, IStates> {
 
         this._lastSelectedPropFieldId = undefined;
       }
+      if (
+        localState &&
+        localState.selected?.emitter &&
+        localState.selected.emitter !== this.state.id &&
+        localState.selected?.value
```

Comment:
Can't the selected value be `undefined` if there is no selection ?

Reply:
Yes it can, do you have a concern about this if condition?

Reply:
I might be missing something, but my understanding is:
`localState.selected?.value` is undefined -> `newState` is not updated -> the `setState` is not triggered -> the panel is not updated
But we probably want to update it to an empty panel since there is no selection.